pkg/gofr/external_db_test.go

```diff
@@ -1,8 +1,11 @@
 package gofr
 
 import (
+	"strconv"
 	"testing"
 
+	"gofr.dev/pkg/gofr/testutil"
```

Comment:
What is your IDE?

I would like to help you setting it up.

I'm not sarcastic, I do want to help

Reply:
sorted the imports ...